Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md

```diff
@@ -549,6 +596,10 @@ Modifies the computer organizational unit within a domain.
 #### Base Command
 
 `ad-modify-computer-ou`
+
+##### Required Permissions
+Requires `Full Control` permissions from `Computer objects`.
```

Comment:
same here 
https://learn.microsoft.com/en-us/answers/questions/973272/delegate-help-desk-users-permission-to-move-users